lib/Doctrine/ORM/Tools/Console/Command/ConvertMappingCommand.php

```diff
@@ -43,6 +43,9 @@
  */
 class ConvertMappingCommand extends Command
 {
+    /** @var string */
+    protected static $defaultName = 'orm:convert-mapping';
```

Comment:
As a minimal version of `symfony/console` dependency in `composer.json` is `3.0` and laziness supports introduced in `3.4`, I duplicate the same command name in `$defaultName` and `->setName()`

Reply:
what about re-using `self::$defaultName` in `setName()` then and mention that `setName()` can be removed when dependencies are bumped to 3.4 minimum in a TODO?

Reply:
Applied. Thank you

Reply:
We can bump the dependency too since this is a minor release, makes things much simpler IMHO

Reply:
Applied. Thank you @lcobucci.